lib/train/extras/os_detect_linux.rb

```diff
@@ -16,91 +16,117 @@ module DetectLinux # rubocop:disable Metrics/ModuleLength
+          @platform[:name] = 'linuxmint'
           @platform[:release] = lsb[:release]
         else
-          @platform[:family] = unix_file?('/usr/bin/raspi-config') ? 'raspbian' : 'debian'
+          @platform[:name] = unix_file?('/usr/bin/raspi-config') ? 'raspbian' : 'debian'
           @platform[:release] = raw.chomp
         end
       elsif !(raw = get_config('/etc/parallels-release')).nil?
-        @platform[:family] = redhatish_platform(raw)
+        @platform[:name] = redhatish_platform(raw)
         @platform[:release] = raw[/(\d\.\d\.\d)/, 1]
       elsif !(raw = get_config('/etc/redhat-release')).nil?
         # TODO: Cisco
         # TODO: fully investigate os-release and integrate it;
         # here we just use it for centos
-        if !(osrel = get_config('/etc/os-release')).nil? && osrel =~ /centos/i
-          @platform[:family] = 'centos'
-        else
-          @platform[:family] = redhatish_platform(raw)
-        end
+        @platform[:name] = if !(osrel = get_config('/etc/os-release')).nil? && osrel =~ /centos/i
+                             'centos'
+                           else
+                             redhatish_platform(raw)
+                           end
+
         @platform[:release] = redhatish_version(raw)
       elsif !(raw = get_config('/etc/system-release')).nil?
         # Amazon Linux
-        @platform[:family] = redhatish_platform(raw)
+        @platform[:name] = redhatish_platform(raw)
         @platform[:release] = redhatish_version(raw)
       elsif !(suse = get_config('/etc/SuSE-release')).nil?
         version = suse.scan(/VERSION = (\d+)\nPATCHLEVEL = (\d+)/).flatten.join('.')
         version = suse[/VERSION = ([\d\.]{2,})/, 1] if version == ''
         @platform[:release] = version
-        @platform[:family] = 'suse'
-        @platform[:family] = 'opensuse' if suse =~ /^openSUSE/
+        @platform[:name] =  if suse =~ /^openSUSE/
+                              'opensuse'
+                            else
+                              'suse'
+                            end
       elsif !(raw = get_config('/etc/arch-release')).nil?
-        @platform[:family] = 'arch'
+        @platform[:name] = 'arch'
         # Because this is a rolling release distribution,
         # use the kernel release, ex. 4.1.6-1-ARCH
         @platform[:release] = uname_r
       elsif !(raw = get_config('/etc/slackware-version')).nil?
-        @platform[:family] = 'slackware'
+        @platform[:name] = 'slackware'
         @platform[:release] = raw.scan(/(\d+|\.+)/).join
       elsif !(raw = get_config('/etc/exherbo-release')).nil?
-        @platform[:family] = 'exherbo'
+        @platform[:name] = 'exherbo'
         # Because this is a rolling release distribution,
         # use the kernel release, ex. 4.1.6
         @platform[:release] = uname_r
       elsif !(raw = get_config('/etc/gentoo-release')).nil?
-        @platform[:family] = 'gentoo'
+        @platform[:name] = 'gentoo'
```

Comment:
Should we add the family `gentoo` here as well? same applies for coreos and alpine